src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -384,15 +378,16 @@ private void EmitLoweredConditionalAccessExpression(BoundLoweredConditionalAcces
             object doneLabel = new object();
             LocalDefinition cloneTemp = null;
 
+            var unconstrainedReceiver = !receiverType.IsReferenceType && !receiverType.IsValueType;
+
             // we need a copy if we deal with nonlocal value (to capture the value)
             // or if we have a ref-constrained T (to do box just once) 
             // or if we deal with stack local (reads are destructive)
+            // or if we have default(T) (to do box just once)
```

Comment:
This is not necessary. Storing 'default(T)' into a temp will just result in more code emitted. 
It would still be correct and it is a rather rare case, so it would be ok, just unnecessary.